### PR TITLE
Allow java install by version in cloud config

### PR DESCRIPTION
**Pull request by caseylitton**
@hitz @Bek PR for fixing java install issue.  I'm testing it one more time then removing my ssh key.  If all goes well it should be ready for use shortly.

### CODE REVIEW

File: cloud-config.yml [hunk 1]
```diff
@@ -1,6 +1,8 @@
 #cloud-config
 
 # Instance
+ssh_authorized_keys:
+  - ssh-rsa AAAAB3NzaC1yc2EAAAADAQABAAACAQC22U1S/NgbEyIUMdAN+v3xHoEIq6u92kpvjgQwg9h34lBabgH8Fl+7+qqTkEnROtmaZlcyOhlz+hF6LhEdzckRf96N76u4pY2MXKnvjMZc3wP/ni9rL7/gysMrtb9zh8SvR5Q+1wnTk3xBA5vvJQ19rk88s2n8ZJn7a3fxw+deUV702y9olW4WLcYNicAmss+qhmbVjFriLMlCqtCHGx9LgaImDGAxMlvcf/D33rmGdYbOLAkLHZa2mLRBVPLLUKbrvTSH+Uvf8wvWIHyDhsULG+LHVP0MCDxet4/Mnq4KTlPwI74Bkpt+/2ja4SZuc3wJ7zTYB17MiVUO6iJlrOpRrQhHbfv1Ll0OnXUV0Vv8x3R2/JtCUpDyEp9dZtr00w9pfL+hU+jbDMgvgeP9WNVM7bBz68GoQYdQKwQoswBUl7zB7WruKFGrNBTR7e5aia7XQcSl95Feb85vqC9QyXgA80FDEwzTCUIx571E3PLludkDIZZpRzFZ7a6OQ0L+rmz/jKABkivnbXbSaEfXOjqgnL4urfh25+jO92SXtHgx5NC/fAe4p6g75nccMpOnCuQ/W/z+yedM7MvG0eRVUYbmWWC9wazGRLStGqcJEEsQhywENqDumyK1V7ogU/UXTaVTlhIBfoDTIwD0dc2i3NYNdPjPU2j7jHQEjzozBGrbew== caseylitton@gmail.com
 
 apt_sources:
 - source: "ppa:webupd8team/java"
```
Comment (caseylitton) [inline cloud-config.yml]: I don't think we need this

File: cloud-config.yml [hunk 2]
```diff
@@ -1,6 +1,8 @@
 #cloud-config
 
 # Instance
+ssh_authorized_keys:
```
Comment (caseylitton) [inline cloud-config.yml]: Remove

File: cloud-config.yml (hunk 1 above)
Comment (Bek) [reply cloud-config.yml]: correct

**Comment by Bek** [comment]
@hitz, install is still failing. It should show up here when the fix is out: https://www.ubuntuupdates.org/ppa/webupd8_java.

**Comment by caseylitton** [comment]
@hitz I've been checking the oracle-java8-installer regularly.  It it still trying to pull Java8_144 as of Friday Morning.

**Comment by caseylitton** [comment]
@hitz @Bek I verified the package installer has been updated to java8_151

**Pull request closed without merging**